modules/pybind11_protobuf/0.0.0-20240524-1d7a729/patches/module-dot-bazel-dev-dependency.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
+index f0b1190..c774abb 100644
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -73,7 +73,7 @@ use_repo(
+ 
+ #### DEV ONLY DEPENDENCIES BELOW HERE ####
+ 
+-pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip", dev_dependency=True)
++pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+ [
+   pip.parse(
+     hub_name = "pypi",
```

Comment:
I'm not sure whether this is safe, as far as I know this name needs to be globally unique.

Reply:
What do you suggest to choose instead?

Reply:
I searched the other modules in BCR and noticed they just use a unique name per module. I updated this module accordingly.